polaris.shopify.com/src/components/HomePage/HomePage.tsx

```diff
@@ -2,17 +2,22 @@ import { BrowseIcon, FigmaIcon, GitHubIcon, InstallIcon } from "./icons";
 import Container from "../Container";
 import Link from "next/link";
 import { className } from "../../utils/various";
+import { useMedia } from "../../utils/hooks";
 import componentsImage from "./components.png";
 import tokensImage from "./tokens.png";
 import iconsImage from "./icons.png";
 import foundationsImage from "./foundations.png";
+import vsCodeImage from "./vscode.png";
 import Image from "../Image";
 import PageMeta from "../PageMeta";
 import styles from "./HomePage.module.scss";
+import { useMediaQuery } from "@shopify/polaris/build/ts/latest/src/utilities/media-query";
 
 interface Props {}
 
 function HomePage({}: Props) {
+  const useMotion = useMedia("(prefers-reduced-motion: no-preference)");
+  console.log(useMotion);
```

Comment:
Is this a leftover?

I also forget to remove console.logs in this project and I think it's because the eslint has no rules in `polaris.shopify.com`